README.md

```diff
@@ -50,12 +50,16 @@ stack test --pedantic
 
 ### Running HLint
 
-All code in this repository should be as idiomatic as possible, so it is
-highly recommended to run `hlint` on your sources before opening a
-*pull request*.
+All code in this repository should be as idiomatic as possible, so we
+enforce in [Travis-CI] that it returns `No hints` when processed by
```

Comment:
Ops! forgot to remove the brackets.

Should we add a links to https://travis-ci.org/exercism/xhaskell when mentioning Travis? I think we have two references to it.